lib/mayaUsd/nodes/stageNode.cpp

```diff
@@ -112,7 +112,7 @@ MStatus UsdMayaStageNode::compute(const MPlug& plug, MDataBlock& dataBlock)
             UsdStageCacheContext ctx(UsdMayaStageCache::Get(loadAll));
 
             bool targetSession = MGlobal::optionVarIntValue(UsdMayaUtil::convert(
-                                     MayaUsdOptionVars->mayaUsd_ProxyTargetsSessionLayerOnOpen))
+                                     MayaUsdOptionVars->ProxyTargetsSessionLayerOnOpen))
                 == 1;
```

Comment:
Same concern about the style.

Reply:
Addressed in 991bdb68dc8ca260b2174f33b705f8a0675a4b17.